ckanext/canada/i18n/fr/LC_MESSAGES/ckanext-canada.po

```diff
@@ -3366,6 +3372,11 @@ msgstr "Passer au contenu principal"
 msgid "Skip to site information"
 msgstr "Passer aux informations sur le site"
 
+#: ckanext/canada/templates/public/user/confirm_delete.html:8
+#, fuzzy
```

Comment:
@RabiaSajjad I THINK this is the correct french phrase?? maybe just add the missing hyphen and remove the fuzzy?